content/posts/shutdown/shutdown.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "shutdown"
+date: 2022-03-13T01:48:05+03:30
+draft: false
+---
+
+<div dir='rtl'>
+
+### فهرست
+
+- [فهرست](#فهرست)
+- [مقدمه](#مقدمه)
+- [خاموش کردن کامپیوتر](#خاموش-کردن-کامپیوتر)
+- [ریبوت کردن کامپیوتر](#ریبوت-کردن-کامپیوتر)
+- [خاموش شدن کامپیوتر در ساعت مشخص](#خاموش-شدن-کامپیوتر-در-ساعت-مشخص)
+- [ریبوت شدن کامپیوتر در ساعت مشخص](#ریبوت-شدن-کامپیوتر-در-ساعت-مشخص)
+- [لغو کردن عملیات](#لغو-کردن-عملیات)
+- [ارسال پیغام به دیگر کاربران قبل از خاموش یا ریبوت شدن](#ارسال-پیغام-به-دیگر-کاربران-قبل-از-خاموش-یا-ریبوت-شدن)
+- [نمایش مجدد عملیات](#نمایش-مجدد-عملیات)
+- [Author or Authors](#author-or-authors)
+</div>
+
+
+
+
+---
+<div dir='rtl'>
+
+### مقدمه
```

Comment:
what?
حوصله؟
really? this is a command like any other commands, why would you write that?
we use command because we use terminal always, and may you are connected to a server and you don`t access the
GUI !